Client/mods/deathmatch/logic/CClientPlayer.h

```diff
@@ -168,6 +171,8 @@ class CClientPlayer final : public CClientPed
     bool                m_bInNetworkInterruption;
     CElapsedTime        m_TimeSinceNetworkInterruptionEnded;
 
+    std::uint8_t scriptDebugLevel;
```

Comment:
Please, add a class member prefix `m_scriptDebugLevel`

Reply:
In addition, this new class member is better to place under one of the booleans for better packing.

Reply:
Okay